routes/sightings.js

```diff
@@ -0,0 +1,53 @@
+import data from '../data/ufoSightings.json';
+import { queried } from '../utils/filters';
+import { paginate } from '../utils/pagination';
+import { validateQueries } from '../utils/restrictions';
+
+export const list = (req, res) => {
+  if (!validateQueries(req.query)) {
+    return res.status(403).json({
+      status: 'error',
+      message:
+        'The queries you provided are not allowed on this endpoint. Please provdie another query'
```

Comment:
Hmm... maybe it would be nice to create error handlers? or even error handling middleware 🕵🏻